plugins/network-elements/nuage-vsp/src/com/cloud/agent/api/sync/SyncDomainCommand.java

```diff
@@ -0,0 +1,91 @@
+//
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+package com.cloud.agent.api.sync;
+
+import com.cloud.agent.api.Command;
+
+public class SyncDomainCommand extends Command {
+
+    private final String _domainUuid;
+    private final String _domainName;
+    private final String _domainPath;
+    private final boolean _toAdd;
+    private final boolean _toRemove;
+
+    public SyncDomainCommand(String domainUuid, String domainName, String domainPath, boolean toAdd, boolean toRemove) {
+        super();
+        this._domainUuid = domainUuid;
+        this._domainName = domainName;
+        this._domainPath = domainPath;
+        this._toAdd = toAdd;
+        this._toRemove = toRemove;
+    }
+
+    public String getDomainUuid() {
+        return _domainUuid;
+    }
+
+    public String getDomainName() {
+        return _domainName;
+    }
+
+    public String getDomainPath() {
+        return _domainPath;
+    }
+
+    public boolean isToAdd() {
+        return _toAdd;
+    }
+
+    public boolean isToRemove() {
+        return _toRemove;
+    }
+
+    @Override
+    public boolean executeInSequence() {
+        return false;
+    }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
This ``equals`` implementation should include attributes from the ``Command`` super class.